ui/plugins/eu.esdihumboldt.hale.ui.functions.numeric/src/eu/esdihumboldt/hale/ui/functions/numeric/SequentialIDParameterPage.java

```diff
@@ -383,8 +411,20 @@ protected String generateExample() {
 
 		prefix = variables.replaceVariables(prefix);
 		suffix = variables.replaceVariables(suffix);
-		startValue = variables.replaceVariables(startValue);
 
-		return prefix + startValue + suffix;
+		int ID = 0;
+
+		if (startValue != "") {
```

Comment:
In Java you can't check for an empty string like this, this compares the references, not the contents.

You could use something like
`if (startValue != null && !startValue.isEmpty()) {`

Reply:
Thank you, this goes directly in my personal collection ;-) https://app.getguru.com/card/cq8pea7i/Some-tricks-for-newbies